webpack.config.js

```diff
@@ -262,6 +262,9 @@ const webpackConfig = ( env, argv ) => {
 						return content;
 					},
 				} ),
+				new DefinePlugin( {
+					GOOGLESITEKIT_VERSION: fs.readFileSync( path.resolve( __dirname, 'google-site-kit.php' ), 'utf8' ).match( /(?<!GOOGLESITEKIT_VERSION',\s+')\d+\.\d+\.\d+(?=')/ig )[ 0 ] || '',
```

Comment:
A few things here:

1. The key should be `global.GOOGLESITEKIT_VERSION`
1. The regular expression doesn't seem to work and returns incorrect data. We probably need to rework it a little bit and use another approach to extract the version from the PHP file. Leaving it to you to figure out the correct way to do it.
1. The value should be appended with `_`.
1. The value should be JSON encoded using the `JSON.stringify()` function.

Reply:
Hey @eugene-manuilov,

1. Updated.
2. For me the regex does work, let me know if you can't see it passing through collectly.
3. Done.
4. Done.

Reply:
@benbowler the regular expression selects a version from the `GOOGLESITEKIT_PHP_MINIMUM` constant instead of `GOOGLESITEKIT_VERSION`. The following is what I see in the generated `dist/assets/js/googlesitekit-api.xxxx.js` file. As you can see it uses `5.6.0` instead of `1.21.0`.

```js
/**
 * Prefix used for all Site Kit keys.
 *
 * Anything not using this key should not be touched by this library.
 *
 * @since 1.5.0
 * @private
 */
var STORAGE_KEY_PREFIX = 'googlesitekit_' + "5.6.0_";
var defaultOrder = ['sessionStorage', 'localStorage'];
var storageBackend;
var storageOrder = [].concat(defaultOrder);
```

Reply:
Yes I see, updated.

Reply:
Awesome! Thanks!